src/middleware.js

```diff
@@ -176,7 +174,7 @@ function apiMiddleware({ getState }) {
       }
 
       // Process the server response
-      if (res.ok) {
+      if (ok(res)) {
```

Comment:
Lets handle errors thrown from the `ok` function in a try/catch here. Should they be handled like those from `doFetch` above?